dmutils/__init__.py

```diff
@@ -1,6 +1,6 @@
-from . import logging, config, proxy_fix, formats, request_id
```

Comment:
Dropping this import makes me nervous but I've had a quick look around and can't immediately see anywhere else referencing it, though these things can be tricky to track down.

Have you run at least one of the frontend apps with this version of utils pulled in?  Would be good to be 100% sure that nothing is expecting to find `logging` here.